### PR TITLE
fix(tablekit-react): update theme colors

### PR DESCRIPTION
**Pull request by SashaShostyr**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @tablecheck/tablekit-css@2.0.1-canary.139.3225512198.0
  npm install @tablecheck/tablekit-react-select@2.0.1-canary.139.3225512198.0
  npm install @tablecheck/tablekit-react@2.0.1-canary.139.3225512198.0
  # or 
  yarn add @tablecheck/tablekit-css@2.0.1-canary.139.3225512198.0
  yarn add @tablecheck/tablekit-react-select@2.0.1-canary.139.3225512198.0
  yarn add @tablecheck/tablekit-react@2.0.1-canary.139.3225512198.0
  ```
</details>

### CODE REVIEW

File: system/react/src/themeVariables/theme.ts [hunk 1]
```diff
@@ -190,35 +226,37 @@ export const sentimentColorsObject = {
...
 export const effectStyles = css`
-  --elevation-large: 0 0 12px 0 rgba(0, 0, 0, 0.1);
-  --elevation-medium: 0 0 6px 0 rgba(0, 0, 0, 0.1);
-  --elevation-small: 0 0 3px 0 rgba(0, 0, 0, 0.1);
+  --elevation-light-large: 0 0 12px 0 rgba(0, 0, 0, 0.1);
+  --elevation-light-medium: 0 0 6px 0 rgba(0, 0, 0, 0.1);
+  --elevation-light-small: 0 0 3px 0 rgba(0, 0, 0, 0.1);
+  --elevation-dark-large: 0 0 12px 0 rgba(255, 255, 255, 0.1);
+  --elevation-dark-medium: 0 0 6px 0 rgba(255, 255, 255, 0.1);
+  --elevation-dark-small: 0 0 3px 0 rgba(255, 255, 255, 0.1);
```
Comment (SimeonC) [inline system/react/src/themeVariables/theme.ts:244]: We'll need some extra logic here to handle splitting this into the light and dark themes as it's not global anymore.

**Review by SimeonC — CHANGES_REQUESTED** [review]
Thanks for splitting the PRs, changes look good apart from one point.

File: system/react/src/themeVariables/theme.ts (hunk 1 above)
Comment (SashaShostyr) [reply system/react/src/themeVariables/theme.ts]: Updated, rereview please

**Review by SimeonC — APPROVED** [review]
Looks great! Thanks for doing the pr.

**Review by wahidfarid — APPROVED** [review]

**Review by oFrederic — APPROVED** [review]

**Review by oFrederic — APPROVED** [review]

**Pull request merged**